modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,68 @@
+The OADP Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications, using CSI volume snapshots pulled from the object store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+The Data Mover solution uses the Restic option of VolSync.
 
-[NOTE]
-=====
 Data Mover supports backup and restore of CSI volume snapshots only.
 
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
-=====
+In OADP 1.2.x Data Mover `VolumeSnapshotBackups` (VSBs) and `VolumeSnapshotRestores` (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously `InProgress`. After all async plugin operations are completed, the backup is marked as completed.
+
+
+[NOTE]
+====
+The OADP 1.2.x Data Mover is a fully supported Red Hat feature.
+
+The OADP 1.1.x Data Mover is a Technology Preview feature.
+====
+:FeatureName: The OADP 1.1.x Data Mover
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+Red Hat recommends that customers who use OADP 1.2.x Data Mover to back up and restore ODF CephFS volumes, upgrade or install OCP 4.12 for improved performance. OADP Data Mover can leverage CephFS shallow volumes in OCP 4.12+, which based on our testing improves the performance of backup times.
+
+* https://https://issues.redhat.com/browse/RHSTOR-4287[CephFS ROX details]
+* https://https://github.com/ceph/ceph-csi/blob/devel/docs/cephfs-snapshot-backed-volumes.md[Provisioning and mounting CephFS snapshot-backed volumes]
+
+
+//For more information about OADP 1.2.x with CephS [name of topic], see ___.
+
+====
 
 .Prerequisites
 
 * You have verified that the `StorageClass` and `VolumeSnapshotClass` custom resources (CRs) support CSI.
 
 * You have verified that only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
++
+[NOTE]
+====
+In OCP 4.12+ verify that this is the only default `volumeSnapshotClass`.
+====
+
+* You have verified that `deletionPolicy` of the `VolumeSnapshotClass` CR is set to `Retain`.
 
 * You have verified that only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
 
 * You have included the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR.
 
+* You have verified that the `OADP namespace` has the annotation `oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'`.
++
+[NOTE]
+====
+In OADP 1.1.x the above setting is mandatory.
+
+In OADP 1.2.x the privileged-movers setting is not required in most scenarios. The restoring container permissions should be adequate for the Volsync copy. In some user scenarios, there may be permission errors that the `privileged-mover`= `true` setting should resolve.
```

Comment:
s/the privileged-movers setting/the `volsync.backube/privileged-movers='true'` annotation (to match the wording used when adding the annotation)

Any guidance on which user scenarios would benefit from the annotation?